helm-charts/strimzi-kafka-operator/templates/031-RoleBinding-strimzi-cluster-operator-entity-operator-delegation.yaml

```diff
@@ -9,6 +9,13 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
 subjects:
+  {{ if .Values.strimzi.watch_namespaces }}
+  {{ range .Values.strimzi.watch_namespaces }}
+  - kind: ServiceAccount
+    name: strimzi-cluster-operator
+    namespace: {{ . }}
```

Comment:
The service account always has to link to the namespace where the Cluster Operator is running. The service accounts do not exist in the watched namespaces. Instead, the whole `RoleBindings` has to be applied to the watched namespaces. That might be achieved by setting the `watch_namespaces` into `metadata.namespace`. But You can always set there only one namespace at a time. So instead of looping over the segment of the YAML you actually need to deploy the RoleBinding multiple times and I have no clue how to do that in Helm / whether its even possible.

_(This basically applies to all RoleBindings, not just to this one)_

Reply:
Actually, maybe the right ay how to do it is to loop the whole RoleBinding object:

```yaml
# ...
metadata:
  namespace: {{ . }}
# ...
---
```

Normally, multiple resources should be fine in one file, so it might work with Helm. WDYT @seglo ?

Reply:
@scholzj Sounds good to me.  Helm will have no problem with multiple resources in one file.